### PR TITLE
DAOS-6977 swim: use DER_ error codes everywhere in SWIM

### PR DESCRIPTION
**Pull request by dmiter**
To avoid mismatch in error codes switch to use DER_ error codes
everywhere.

### CODE REVIEW

**Review by jolivier23 — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

File: src/cart/crt_swim.c [hunk 1]
```diff
@@ -332,16 +331,16 @@ static int crt_swim_send_message(struct swim_context *ctx, swim_id_t to,
 
 	rc = crt_req_create(crt_ctx, &ep, opc, &rpc_req);
 	if (rc) {
-		D_ERROR("crt_req_create() failed "DF_RC"\n", DP_RC(rc));
+		D_ERROR("crt_req_create(): "DF_RC"\n", DP_RC(rc));
```
Comment (ashleypittman) [inline src/cart/crt_swim.c:334]: We need to settle on a standard format for this as there's a lot of variation and changes in format and it's not always in the same direction.

**Comment by dmiter** [comment]
Can we land this?

File: src/cart/crt_swim.c (hunk 1 above)
Comment (dmiter) [reply src/cart/crt_swim.c:334]: I'm fine with any...

**Comment by johannlombardi** [comment]
it needs to merge master.

**Comment by dmiter** [comment]
It's ready for review and then landing...

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**